index.html

```diff
@@ -42,7 +42,7 @@
 </head>
 <body>
   <section class="site-banner">
-    <span>Grain <a href="/blog/2021/08/16/grain-awarded-50000-grant-from-near-foundation/" target="_blank" rel="noopener">awarded $50,000 grant</a> from NEAR Foundation</span>
+    <span>Say hello to <a href="/blog/2021/09/07/new-release-grain-v0.4-cereal/" target="_blank" rel="noopener">Grain v0.4, Cereal</a>—With support for the NEAR Protocol & improved tooling!</span>
```

Comment:
```suggestion
    <span>New release <a href="/blog/2021/09/07/new-release-grain-v0.4-cereal/" target="_blank" rel="noopener">Grain v0.4 Cereal</a>—new platforms & improved tooling</span>
```

Reply:
Should there be a `:` between "New Release" or something? Does it look a little weird? (I kinda liked the "Say hello" part, but doesn't need to stay)

Reply:
Honestly wanted to look at it in the preview and decide 😛

Reply:
Or say hello is fine!